packages/beeq/src/components/progress/scss/bq-progress.variables.scss

```diff
@@ -0,0 +1,13 @@
+/* -------------------------------------------------------------------------- */
+/*                   Progress custom properties                  */
+/* -------------------------------------------------------------------------- */
+
+:host {
+  /**
+   * @prop --bq-progress--width - Width of the progress
+   * @prop --bq-progress-left-offset-position - Default left offset tooltip position
+   */
+
+  --bq-progress--width: 250px;
```

Comment:
We should let the component to adjust to his parent width rather than forcing the width.